app/assets/javascripts/backbone/views/editable_breakeven_table_view.js.coffee

```diff
@@ -0,0 +1,27 @@
+class MS.Views.EditableBreakevenTableView extends Backbone.View
```

Comment:
I was unable to get `extends MS.Views.EditableTableView` working here. It is mostly duplicated from `EditableTableView`